libraries/AP_Frsky_Telem/AP_Frsky_Telem.h

```diff
@@ -15,14 +15,28 @@
 #pragma once
 
 #include <AP_HAL/AP_HAL.h>
-#include <AP_Notify/AP_Notify.h>
-#include <AP_SerialManager/AP_SerialManager.h>
 #include <AP_HAL/utility/RingBuffer.h>
+#include <AP_Notify/AP_Notify.h>
 #include <AP_RCTelemetry/AP_RCTelemetry.h>
+#include <AP_SerialManager/AP_SerialManager.h>
 
-#define FRSKY_TELEM_PAYLOAD_STATUS_CAPACITY          5 // size of the message buffer queue (max number of messages waiting to be sent)
+#include "frsky.h"
+
+#if HAL_WITH_FRSKY_TELEM_BIDIRECTIONAL
+#include "mavlite.h"
+using namespace MAVLITE;
+// for fair scheduler
+#define TIME_SLOT_MAX               12U
```

Comment:
These defines are available down in GCS_MAVLink.  I don't think we really want them there - especially not with such generic names.